aiqleads/scripts/__init__.py

```
@@ -1 +1,12 @@
+"""Scripts package for running various project tools.
+
+This package contains scripts for:
+- Testing and validation
+- Project status management
+- Development utilities
+"""
+
+from .run import test_project_tracking
+from .test_runner import run_tests
+
+__all__ = ['test_project_tracking', 'run_tests']
```

Ruff (0.8.2)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)